src/command_helpers/configure_command.rb

```diff
@@ -40,11 +40,15 @@ class ConfigureCommand < BaseCommand
       EOF
 
       def run
-        configurator.configure
+        configurator.configure(answers)
         custom_configuration
         render_domain_templates
       end
 
+      def answers
+        JSON.parse(options.answers) if options.answers
```

Comment:
Could we change this to explicitly return `nil` using a ternary operator? The `nil` is important behaviour for how `configure` works so making it return explicitly would be helpful.